sqlx-postgres/src/types/mod.rs

```diff
@@ -13,6 +13,7 @@
 //! | `f64`                                 | DOUBLE PRECISION, FLOAT8                             |
 //! | `&str`, [`String`]                    | VARCHAR, CHAR(N), TEXT, NAME, CITEXT                 |
 //! | `&[u8]`, `Vec<u8>`                    | BYTEA                                                |
+//! | `BTreeMap<String, Option<String>>`    | HSTORE                                               |
```

Comment:
As `PgHStore`, this should go down below with the rest of the `PgFoo` types.

Reply:
That makes perfect sense, changes applied as requested.